### PR TITLE
Upload and manuplate configurable forms

### PR DESCRIPTION
**Pull request by ciremusyoka**
Addresses issue https://github.com/onaio/reveal-frontend/issues/900 

performs the following tasks:
-Upload config form
-Edit config form
-Download uploaded config forms
-Publish manifest files

### CODE REVIEW

**Comment by ciremusyoka** [comment]
Screenshot from 2020-06-19 18-48-13
Screenshot from 2020-06-19 18-48-40
Screenshot from 2020-06-19 18-48-50
Screenshot from 2020-06-19 18-49-07
Screenshot from 2020-06-19 18-49-49
Screenshot from 2020-06-19 18-53-11

File: clients/core/src/App/App.tsx [hunk 1]
```diff
@@ -71,6 +119,7 @@ class App extends Component {
                                     />
                                 )}
                             />
+                            ManifestReleases
```
Comment (p-netm) [inline clients/core/src/App/App.tsx:122]: I don't think this should be here.

File: clients/core/src/constants.tsx [hunk 2]
```diff
@@ -66,6 +82,78 @@ export const SIGN_OUT = 'Sign Out';
...
+export type JSON_VALIDATOR_LABLE = typeof JSON_VALIDATOR_LABLE;
+export const DRAFT_FILE_LABEL = 'Draft Files';
+export type DRAFT_FILE_LABEL = typeof DRAFT_FILE_LABEL;
+export const RELEASE_FILE_LABEL = 'Release';
+export type RELEASE_FILE_LABEL = typeof RELEASE_FILE_LABEL;
+export const RELEASES_LABEL = 'Releases';
+export type RELEASES_LABEL = typeof RELEASES_LABEL;
+export const EDIT_FILE_LABEL = 'Edit Form';
+export type EDIT_FILE_LABEL = typeof EDIT_FILE_LABEL;
+export const UPLOAD_FILE_LABLE = 'Upload Form';
```
Comment (p-netm) [inline clients/core/src/constants.tsx:105]: nitpick: a few misspelled vars `LABLE`

File: clients/core/src/containers/pages/FormConfig/manifest/uploadFile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        validatorsUrl: VIEW_JSON_VALIDATORS_PAGE_URL,
+    };
+
+    const titleAction = formId ? EDIT_FILE_LABEL : UPLOAD_FILE_LABEL;
+    return (
+        <div>
+            <Helmet>
+                <title>{titleAction}</title>
+            </Helmet>
+            <h3 className="household-title">{titleAction}</h3>
```
Comment (p-netm) [inline clients/core/src/containers/pages/FormConfig/manifest/uploadFile/index.tsx:47]: @ciremusyoka should we rename this class name to sth more generic.

**Pull request merged**